### DISCUSSION TITLE
<Line  /> BackgroundColor does not change when it's DynamicResource value changes ?

### DISCUSSION THREAD

**Discussion by softlion** [dotnet/maui#14904] [General]
I have this simple code to define a separator line in a `CollectionView`:

```xaml
<Style TargetType="Line" x:Key="SeparatorLine">
     <Setter Property="HeightRequest" Value="1" />
     <Setter Property="BackgroundColor" Value="{DynamicResource LineColor}" />
</Style>
```

That I use like this:
```xaml
<Line Style="{StaticResource SeparatorLine}" />
```

The LineColor is changing when the color theme is dynamically changed.
```xaml
<Color x:Key="LineColor">#eeeeee</Color>
or
<Color x:Key="LineColor">#222</Color>
```

But the color of the line does not change.  
I have other controls using the same system where the color is correctly changing.  
Is that a bug in maui ?

**Comment by logikonline** [comment]
It will also fail using the AppThemeBinding as well.

`<Line HorizontalOptions="FillAndExpand" BackgroundColor="{AppThemeBinding Dark={StaticResource DarkSeparatorColor}, Light={StaticResource LightSeparatorColor}}" Margin="20,0,20,0" Grid.Row="2"/>`
